recipes/parsl/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "parsl" %}
+{% set version = "0.7.2" %}
+{% set sha256 = "4fe2df4d2e49aaa6f03eab6a3798f25882cd08253df2246691a95a24ab9f7db0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Parsl/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not linux or py<35]
```

This looks like it could be a noarch package:
```suggestion
  noarch: python
```

---

I just noticed you have `entry_points` so you'll also need to add those. [See here](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html?highlight=selectors#python-entry-points)

---

> noarch: python

Hi Chris, Thanks for the help. I am a bit confused here. Parsl has a focus on linux and has not been tested on win or mac osx -- that is why I place a selector here.  According to the [example receipe](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml) and the automated linter suggestion above, it seems `noarch python`  should not be used here?

---

> I just noticed you have `entry_points` so you'll also need to add those. [See here](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html?highlight=selectors#python-entry-points)

@chrisburr Thanks. I have added the entry_points.

---

`noarch: python` packages can't use the selector syntax (`# [py<35]`) but they can limit the Python version using `python >=3.5` in the `host` and `run` dependencies. The two methods of limiting the Python version should not be used together.

The advantage of this being a noarch package is that there will be only one build (rather than one build per Python version per platform). There are noarch packages with dependencies that only exist on Linux and that's not a problem (but it does cause errors in the staged-recipes CI which can be ignored).

I guess the question is do Windows and macOS users need to be explicitly forbidden from installing this package? Or can it be an unsupported configuration as is the case for your PyPI package?